benchmark_v2/dataframe_indexing_benchmark.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+import numpy as np
+import pandas as pd
+import pytest
+
+import arkouda as ak
+
+OPS = ["_get_head_tail_server", "_get_head_tail"]
+
+
+def generate_dataframe():
+    cfg = ak.get_config()
+    N = pytest.prob_size * cfg["numLocales"]
+    types = [ak.Categorical, ak.pdarray, ak.Strings, ak.SegArray]
+
+    # generate random columns to build dataframe
+    df_dict = {}
+    np.random.seed(pytest.seed)
+    for x in range(20):  # loop to create 20 random columns
+        key = f"c_{x}"
+        d = types[x % 4]
+        if d == ak.Categorical:
+            str_arr = ak.random_strings_uniform(minlen=5, maxlen=6, size=N, seed=pytest.seed)
+            df_dict[key] = ak.Categorical(str_arr)
+        elif d == ak.pdarray:
+            df_dict[key] = ak.array(np.random.randint(0, 2**32, N))
+        elif d == ak.Strings:
+            df_dict[key] = ak.random_strings_uniform(minlen=5, maxlen=6, size=N, seed=pytest.seed)
+        elif d == ak.SegArray:
+            df_dict[key] = ak.segarray(
+                ak.arange(0, N), ak.array(np.random.randint(0, 2**32, N))

```

Comment:
Should we keep this `ak.arange(0, N, 5)`? That way we have more than one segment in our segarray

Reply:
I'm not opposed to adding the step of 5 back. But just to clarify what this is doing, it is increasing the size of each segment to 5 values per segment instead of 1 value per segment.

Reply:
Ah yes! you're right, my bad

Reply:
After further discussion with @Ethan-DeBandi99 and @pierce314159, we've decided to leave this as it is for now